src/Microsoft.DotNet.Interactive.CSharpProject/RoslynWorkspaceUtilities/RoslynWorkspaceUtilities.cs

```diff
@@ -172,8 +172,7 @@ public static CodeAnalysis.Project AddToWorkspace(BuildProjectData analyzerResul
         ProjectInfo projectInfo = GetProjectInfo(analyzerResult, workspace, projectId);
         if (projectInfo is null)
         {
-            // Error
-            return null;
+            throw new ArgumentNullException(nameof(projectInfo));
```

Comment:
This isn't the correct exception type, since it's not an argument.

What causes `projectInfo` to be null? Is it due to a bug somewhere else?

Reply:
I'll push a cleanup pr to fix this